doc/source/install/install.md

```diff
@@ -253,39 +255,61 @@ memory pages are to be defined.
 
 ### 4.1 Server Provisioning
 
+Note:  All of the following steps are executed within the root directory of your 
+snaps-boot project.  This will usually be snaps-boot/, so it assumed you are in this 
+directory. 
+
 #### Step 1
 
+##### PXE Bios Install
 Download `ubuntu16.04 server image` from internet and need to place it
-in folder `snaps-boot/packages/images/`. Use this download
+in folder `packages/images/`. Use this download
 link for ISO:
  http://releases.ubuntu.com/16.04/ubuntu-16.04.4-server-amd64.iso.
 
 ```
-cd snaps-boot/
 mkdir -p packages/images
 cd packages/images
 wget http://releases.ubuntu.com/16.04/ubuntu-16.04.4-server-amd64.iso
 ```
 
+##### PXE UEFI Install
```

Comment:
Could we automate the downloading of these files?

Reply:
https://github.com/cablelabs/snaps-boot/issues/112